test/commons/dom/has-lang-text.js

```diff
@@ -0,0 +1,83 @@
+describe('dom.hasLangText', function() {
+  'use strict';
+  var hasLangText = axe.commons.dom.hasLangText;
+  var fixture = document.getElementById('fixture');
+  var tree;
+
+  it('returns true when the element has a non-empty text node as its content', function () {
+    fixture.innerHTML = '<div id="target">  text  </div>';
+    tree = axe.utils.getFlattenedTree(fixture);
+    var target = axe.utils.querySelectorAll(tree, '#target')[0];
+    assert.isTrue(hasLangText(target, { skipNestedLang: true }));
```

Comment:
I don't see `skipNestedLang` as an option for `hasLangText`. Should it be here for all the function calls?

Reply:
Good spot. This is a holdover from something I tried to do before.